terraform/k8s/providers.tf

```diff
@@ -0,0 +1,24 @@
+variable "config_context" {
```

Comment:
You should update this to be a [local value](https://developer.hashicorp.com/terraform/language/values/locals)


```
locals {
  config_context = "docker-desktop"
}
```

and reference `local.config_context`